resources/mysqlbinlog/mysqlbinlog_8.0.28-macos11-arm64_test.go

```diff
@@ -0,0 +1,43 @@
+//go:build darwin && arm64
```

Comment:
How about renaming this file to `darwin_test.go`, because the test logic is not relevant to a specific version of `mysqlbinlog`, and the package is already named "mysqlbinlog".

Reply:
Follow https://github.com/bytebase/bytebase/pull/1266#discussion_r870124309